v1/lib/v1/searchable.rb

```diff
@@ -84,6 +87,9 @@ def search(params={})
 
     def search_offset(params)
       page = params["page"].to_i
+      if page > MAX_PAGE_NUM
+        raise BadRequestSearchError, "Page value #{page} is too high"
```

Comment:
will this crash the app or will it just produce a 404?

Reply:
It just causes a 400 Bad Request response to the client.

Reply:
ok so if this doesnt kill the frontend then i'd say it's good to go